plugins/shodan/api_key.go

```diff
@@ -0,0 +1,57 @@
+package shodan
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://developer.shodan.io/api/requirements"),
```

Comment:
This looks to refer to the API, could we replace this by https://cli.shodan.io/?

Reply:
I replaced the link with the one you provided.